src/accessories/SearchAssistant.ttslua

```diff
@@ -130,7 +135,31 @@ function startSearch(messageColor, number)
   setAsideRotation = { handData.rotation.x, handData.rotation.y + 180, 180 }
 
   -- place hand cards set aside
-  deckLib.placeOrMergeIntoDeck(handCards, setAsidePosition, setAsideRotation)
+  if #handCards > 0 then
+    deckLib.placeOrMergeIntoDeck(handCards, setAsidePosition, setAsideRotation)
+  else
+    oldVectorLines = Global.getVectorLines()
+    lines = {}
+    local pos = setAsidePosition
+    if matColor == "Green" or matColor == "Red" then
+      pointsLine1 = { { pos.x - 0.5, pos.y, pos.z + 1 }, { pos.x + 0.5, pos.y, pos.z - 1 } }
+      pointsLine2 = { { pos.x - 0.5, pos.y, pos.z - 1 }, { pos.x + 0.5, pos.y, pos.z + 1 } }
+    else
+      pointsLine1 = { { pos.x - 1, pos.y, pos.z + 0.5 }, { pos.x + 1, pos.y, pos.z - 0.5 } }
+      pointsLine2 = { { pos.x - 1, pos.y, pos.z - 0.5 }, { pos.x + 1, pos.y, pos.z + 0.5 } }
```

Comment:
This can very likely be simplified by using setAsideRotation

Reply:
I tried this first, but I don't think the vector lines rotate in quite the same way as the existing variable.